modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -1,130 +1,109 @@
-http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
-http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/mule/ee/tracking/current/mule-tracking-ee.xsd">
+To use the scheduling feature in Runtime Manager, deploy an application that includes one or more Scheduler elements: 
 
-    <flow name="synchronize-accounts" doc:name="synchronize-accounts">
-        <poll frequency="300000" doc:name="Poll">
-            <logger level="INFO" doc:name="Start" message="Start Account Synchronization ${salesforce.username}"/>
-        </poll>
-        <logger level="INFO" doc:name="End" message="End Account Synchronization"/>
-    </flow>
-    <flow name="synchronize-contacts" doc:name="synchronize-contacts">
-        <poll frequency="600000" doc:name="Poll">
-            <logger level="INFO" doc:name="Start" message="Start Contact Synchronization ${salesforce.username}"/>
-        </poll>
-        <logger level="INFO" doc:name="End" message="End Contact Synchronization"/>
-    </flow>
-</mule>
-----
-=====
+.The arrow shows the *Scheduler* component in the Anypoint Studio canvas.
+image::cloudhub-scheduler-studio.png[Scheduler component in the Anypoint Studio canvas]
 
-CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes, so that you don't need to perform any other special configuration.
+You can download this example app from Exchange.
 
-After deploying an app, you can view the polling elements in Runtime Manager by selecting the app and clicking *Schedules* in the left navigation menu. Runtime Manager lists the polling elements in your application in the order they appear in your application:
-
-image::ch-view-polling-schedule.png[View Schedules]
-
-The name of each element is determined by the `doc:name` attribute in the polling scopes, if specified in the XML. If you do not specify `doc:name`, CloudHub applies the name, *yourFlowname Poll*.
-
-The *Last Run* column shows the date and time each polling element last ran. You can view the logs to see more information about when the jobs started and ended scheduled runs.
-
-== Manage Schedules at Runtime
-
-To manage schedules in Runtime Manager:
+After deploying an app to CloudHub, view and manage the Scheduler elements in Runtime Manager:
 
 . Click *Applications* in the left pane, click the app, and then click *Schedules*:
 +
-image::ch-schedule-change.png[View Schedules]
+.The screenshot shows (*1*) the *Logs* option, (*2*) the *Last Run* column, and (*3*) the frequency
+link on the *Schedules* page.
```

Comment:
Is it okay that this is not blue?

Reply:
Good catch! The image wasn't showing, too.